_studio/shared/src/mfx_umc_alloc_wrapper.cpp

```diff
@@ -384,7 +387,9 @@ void mfx_UMC_FrameAllocator::SetExternalFramesResponse(mfxFrameAllocResponse *re
 {
     m_externalFramesResponse = 0;
 
-    if (!response || !response->NumFrameActual)
+    if (!response)
+        return;
+    if (!response->NumFrameActual && !m_lazyRegistration)
```

Comment:
May it makes sense to add check like
`if (response->NumFrameActual && m_lazyRegistration)`

PS I'd rather add `assert` (or even `throw std::logic_error`) before `return`

Reply:
done.